App/Screens/Checking/Lists.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+
+import { View } from 'react-native-animatable';
+import { ListItem, Avatar } from 'react-native-elements';
+import styles from './style';
+
+export const dollars11 = [63, 17, 1200, 320];
+export const cents11 = [95, 75, 50, 73];
+
+export const list11 = [
+	{
+		title: 'Target',
+		subtitle: 'Closter NJ | Debit card',
+		dollars: `$${dollars11[0]}`,
+		cents: `.${cents11[0]}`,
+	},
+	{
+		title: 'AplPay 7-Eleven',
+		subtitle: 'Cresskill NJ | iPhone',
+		dollars: `$${dollars11[1]}`,
+		cents: `.${cents11[1]}`,
+	},
+	{
+		title: 'Facebook inc',
+		subtitle: 'Pay day! Yay!',
+		dollars: `$${dollars11[2]}`,
+		cents: `.${cents11[2]}`,
+	},
+	{
+		title: 'Lencrafters',
+		subtitle: 'Paramus NJ | Debit card',
+		dollars: `$${dollars11[3]}`,
+		cents: `.${cents11[3]}`,
+	},
+];
+
+export const dollars10 = [10000, 12, 236, 320];
+export const cents10 = [0, 2, 52, 73];
+
+export const list10 = [
+	{
+		title: 'Transfer from savings',
+		subtitle: 'Buy a house (...4044)',
+		dollars: `$${dollars10[0]}`,
+		cents: `.${cents10[0]}`,
+	},
+	{
+		title: 'Starbucks',
+		subtitle: 'Closter NJ | Debit card',
+		dollars: `$${dollars10[1]}`,
+		cents: `.${cents10[1]}`,
+	},
+	{
+		title: 'Stop and Shop',
+		subtitle: 'Closter NJ | Debit card',
+		dollars: `$${dollars10[2]}`,
+		cents: `.${cents10[2]}`,
+	},
+	{
+		title: 'Lencrafters',
+		subtitle: 'Paramus NJ | Debit card',
+		dollars: `$${dollars10[3]}`,
+		cents: `.${cents10[3]}`,
+	},
+];
+
+export const ListOfJul11 = () => {
+return (
+	<View style={{ borderRadius: 20 }}>
+		{list11.map((item, i) => (
+      <ListItem key={i} bottomDivider containerStyle={{borderRadius: 10}}>
+        {i == 0 }
```

Comment:
```suggestion
        {i === 0 }
```
do not use `==` at all, use `===` instead every time